packages/client/src/comp/Status.tsx

```diff
@@ -27,7 +29,7 @@ export const Status = () => {
     <ul>
       <li>NODE_ENV: {process.env.NODE_ENV}</li>
       <li>REACT_APP_ENVIRONMENT: {process.env.REACT_APP_ENVIRONMENT}</li>
-      <li>REACT_APP_EMAIL_FAX_OFFICIALS: {process.env.REACT_APP_EMAIL_FAX_OFFICIALS}</li>
+      <li>REACT_APP_EMAIL_FAX_OFFICIALS: {featureFlags?.REACT_APP_EMAIL_FAX_OFFICIALS}</li>
```

Comment:
Now that we aren't using the env var in the front end, let's call it `EMAIL_FAX_OFFICIALS`

Reply:
done